libdnf-cli/output/repoqueryformat.cpp

```diff
@@ -223,4 +225,33 @@ void print_pkg_set_with_format(
     }
 }
 
+void print_pkg_attr_uniq_sorted(
+    std::FILE * target, const libdnf::rpm::PackageSet & pkgs, const std::string & getter_name) {
+    auto getter = NAME_TO_GETTER.find(getter_name);
+    if (getter == NAME_TO_GETTER.end()) {
+        throw RuntimeError(M_("package getter: %s not available"), getter_name);
+    }
+    std::set<std::string> output;
+    for (auto package : pkgs) {
+        std::visit(
+            [&output, &package](auto && getter_func) {
```

Comment:
Why is rvalue reference needed here? Wouldn't `(const auto & getter_func)` do?

Reply:
Right, updated, thanks. :+1: